pipeline_steps/delivery.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+import pytz, logging
+from base import PipelineStep
+class PipelineDelivery(PipelineStep):
+    @staticmethod
+    def process_message(message):
+        logging.warning(message.current_step + " has received " + message.name + " " + str(message.id))
+        if pytz.utc.localize(datetime.utcnow()) > message.expiration_time:
+            logging.warning(message.name + " " + str(message.id) + " has expired. It will be dropped.")
```

Comment:
Yeah.  Even though making this a regular instance method isn't required at the moment, it's probably a good idea, just to keep things consistent, since there's no reason to forbid it from accessing member attributes, if they existed.  But if you do, don't forget to add a self argument to the definition.